packages/cf-component-tabs/src/Tabs.js

```diff
@@ -13,6 +14,85 @@ const find = (list, condition) => {
   return foundElement;
 };
 
+const styles = ({ theme }) => ({
+  marginTop: '1.5rem',
+  marginBottom: '1.5rem',
+  border: '1px solid #dedede'
```

Comment:
I couldn't find any `#dedede` color but I'll do a manual comparison and find a close one for this whole PR.